.luacheckrc

```diff
@@ -1,6 +1,10 @@
 globals = {
     "_",
-    "vim"
+    "vim",
+    "describe",
+    "it",
+    "assert",
+    "stup",
```

Comment:
I'm not sure why this was added to the globals, maybe it was a typo? I couldn't find anything with that name in the code